packages/angular/test/base/e2e/src/standalone/tabs.spec.ts

```diff
@@ -13,4 +13,8 @@ describe('Tabs', () => {
     cy.get('app-tab-two').should('be.visible');
     cy.contains('Tab 2');
   });
+
+  it('parentOutlet should be defined', () => {
```

Comment:
Suggestion: Add an "annotation" referencing the issue that lead to the creation of this test.